index.js

```diff
@@ -1,150 +1,180 @@
-/* jshint node: true */
-'use strict';
+/* eslint-env node */
+"use strict";
```

Comment:
If this is prettified, we should copy over our prettier config. This looks like a lot of changes that we normally don't do

Reply:
Great idea! Done.